go.mod

```diff
@@ -31,3 +31,5 @@ require (
 	google.golang.org/grpc v1.28.1
 	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
 )
+
+replace github.com/braintree/heckler => ../heckler
```

Comment:
did you mean to commit this change or was this just for testing?

Reply:
"replace github.com/braintree/heckler => ../heckler"  for this commit I will remove it. We are adding a Change Management feature(For SNOW/JIRA/3rd party Tickets systems) in the next PRs. I will add it later.